account_analytic_parent/models/account_analytic_account.py

```diff
@@ -44,7 +59,7 @@ def _compute_debit_credit_balance(self):
                 groupby=['account_id']
             )
             data_debit = sum(l['amount'] for l in debit_groups)
-            account.debit = data_debit
+            account.debit = abs(data_debit)
```

Comment:
just in debit?

Reply:
It was in original PR, thus question to @cubells I believe

Reply:
Btw, is this needed at all? If there are 10 accounts with -100 debit, total is -1000, not 1000. Or I'm missing something?

Reply:
No debit, should be positive always (as the accounting)

Reply:
No, you are putting as positive the total sum, to see 1000 instead of -1000. Take into account that we split debit/credit from an unique value: amount. Example, if amount == -100 (< 0 in general), then amount is assigned to debit. If amount == 100 (> 0 in general), then amount is assigned to credit. The other option is to assign `-amount` to debit in that case.